app/star-rating.sass

```diff
@@ -9,4 +9,32 @@ h1
    margin: 0
 
 #app
-   font-size: 3em
\ No newline at end of file
+   font-size: 3em
+
+.star-rating
+    display: inline-block
+    unicode-bidi: bidi-override
+    direction: rtl
+    border-width: 0px
```

Comment:
Can be simplified to `border: 0` or `border: none`

Reply:
See commit "simplifies star-rating border style"